collector/cpu_freebsd.go

```diff
@@ -0,0 +1,73 @@
+// +build !nostat
+
+package collector
+
+import (
+	"errors"
+	"os"
+	"strconv"
+	"unsafe"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+/*
+#cgo LDFLAGS: -lkvm
+#include <fcntl.h>
+#include <kvm.h>
+#include <sys/param.h>
+#include <sys/pcpu.h>
+#include <sys/resource.h>
+*/
+import "C"
+
+const ()
+
+type statCollector struct {
+	cpu *prometheus.CounterVec
+}
+
+func init() {
+	Factories["cpu"] = NewStatCollector
+}
+
+// cpu stats.
```

Comment:
Also, please make this a full sentence of what this method does (see e.g. https://github.com/prometheus/node_exporter/blob/master/collector/filesystem.go#L37-L38). Same for the other collectors.